ehrql/tables/beta/tpp.py

```diff
@@ -251,60 +251,142 @@ class ec_cost(EventFrame):
     ec_ident = Series(
         int,
         constraints=[Constraint.NotNull()],
-        description="TODO",
+        description="Primary key used to join to the other EC tables",
     )
     grand_total_payment_mff = Series(
         float,
-        description="TODO",
+        description="The total payment for the activity (Net_SLA_Payment + Tariff_MFF_Payment).",
     )
     tariff_total_payment = Series(
         float,
-        description="TODO",
+        description="The total payment according to the national tariff.",
     )
     arrival_date = Series(
         datetime.date,
-        description="TODO",
+        description="The date the patient self presented at the accident & emergency department, or arrived in an ambulance at the accident & emergency department.",
     )
     ec_decision_to_admit_date = Series(
         datetime.date,
-        description="TODO",
+        description="The date a decision to admit was made.",
     )
     ec_injury_date = Series(
         datetime.date,
-        description="TODO",
+        description="The date the patient was injured.",
     )
 
 
 @table
 class emergency_care_attendances(EventFrame):
-    id = Series(int)  # noqa: A003
-    arrival_date = Series(datetime.date)
-    discharge_destination = Series(SNOMEDCTCode)
+    id = Series(  # noqa: A003
```

Comment:
question: Should `id` be `ec_ident` for consistency?

Reply:
I also think this should be done, but think it would be better as a separate PR, as with renaming the `emergency_care_attendances` table.

These would be breaking changes, unless we provide aliases for the old names, whereas the current PR can be kept to non-breaking documentation additions.